xlators/mgmt/glusterd/src/glusterd-handshake.c

```diff
@@ -1916,16 +1912,13 @@ glusterd_event_connected_inject(glusterd_peerctx_t *peerctx)
 
     ret = glusterd_friend_sm_inject_event(event);
```

Comment:
I would say that we don't need to do this inside the rcu. `peerinfo` is not used anymore.